mod-tool/src/main/scala/at/forsyte/apalache/tla/Tool.scala

```diff
@@ -319,7 +320,7 @@ object Tool extends LazyLogging {
     runAndExit(
         executor,
         _ => "No example found",
-        "Checker has found an example. Check counterexample.tla.",
+        "Found a violation of the postcondition. Check failure.tla.",
```

Comment:
Why are we renaming counterexample to `failure`? This doesn't seem easier to understand to me, and seems to be an inaccurate name given the contents of the file.

Reply:
It has been always hard to explain why counterexamples are "counter"-examples. Also, the word is too long :)

 Now I wanted to add examples and this makes things even more confusing. I don't like the word failure either, but as Jure pointed out, "error" would not be technically correct. Perhaps, a "violation" would be a better term.

Reply:
ok, then I will change it to "violation" and merge this PR later